docs/cross-platform-development.md

```diff
@@ -0,0 +1,3 @@
+## Writing cross-platform apps
```

Comment:
Do you want it to be along the lines of "writing for Windows"? We have projects going on with Ember Cli and Electron - I could contribute a ton of stuff around calling Win32 from Node, how to deal with path differences across platforms (`/` vs `\`), etc. 

In that case, we could just change the title to cross-platform-tips.